.changelog/16204.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: update Ember version to 3.28 LTS;
```

Comment:
CI is complaining here because the changelog file number has to match the PR number, so moving this to `.changelog/16227.txt` should fix this.

One of the devs on the core team put together [this tool](https://github.com/kisunji/gen-changelog) to make generating changelogs easier.

Reply:
ah you are right! I've copied this one from url with your pr, which I used as example for description 🙃